public/stylesheets/main.css

```diff
@@ -1,108 +1,59 @@
-html,
-body {
-    padding: 0;
```

Comment:
html, body padding 0 and margin 0 must be added here to html and body. 

i'm not gonna redo these styles back. read please why they are needed here. also box-sizing border box.

there styles are must in every project as defaults.